src/App.js

```diff
@@ -4,13 +4,16 @@ import AddTodoForm from './AddTodoForm';
 import TodoList from './TodoList';
 
 function App() {
-  const [newTodo, setNewTodo] = useState(); //use state hook
+  const addTodo = (newTodo) =>{
+    setTodoList([...todoList, newTodo])
```

Comment:
Doesn't requires changes, but highly recommended that it's addressed soon:
While line 8 works as intended, there's a bug in how its implemented. To be brief, React batches state updates. Therefore, if you have something like this:
```js
const [count, setCount] = useState(0);

const addCount = () => {
  setCount(count + 1) // expected 0 + 1
  setCount(count + 1) // expected 1 + 1
  setCount(count + 1) // expected 2 + 1
  // expected count = 3
}
```
You may think the count is being set to 3, but it's not! Instead, since React batches those state setters, it takes the current count (0) and increases it by 1 multiple times:
```js
setCount(count + 1) // actual 0 + 1
setCount(count + 1) // actual 0 + 1
setCount(count + 1) // actual 0 + 1
// actual count = 1
```

To mitigate this bug, you'll want to use a **function** inside of the state setter whose first argument is the previous state (`prevCount`) and returns the next state (`prevCount + 1`):
```js
setCount(prevCount => prevCount + 1) // actual 0 + 1
setCount(prevCount => prevCount + 1) // actual 1 + 1
setCount(prevCount => prevCount + 1) // actual 2 + 1
// actual count = 3
```
---
With the above knowledge, how might you change this to reflect the above?
```js
setTodoList([...todoList,newTodo])
```

<details>
<summary>Click here for answer if you're stuck</summary>

```js
setTodoList(prevTodoList => ([ ...prevTodoList, newTodo ]))
```
</details>